src/AccessibilityInsights.Extensions.TelemetryTests/Extensions.TelemetryTests.csproj

```diff
@@ -64,6 +68,7 @@
   <ItemGroup>
     <Compile Include="AITelemetryTests.cs" Condition="$(FAKES_SUPPORTED) == 1" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="OSHelpersUnitTests.cs" />
```

Comment:
I don't see us excluding the fakes based on the environment variable anywhere. We can do this here at the file level, or at the function level in the file itself.

Reply:
This was added in the most recent iteration